Update docs to reflect change from nemoretriever-parse to nemotron-parse (#1198)
NVIDIA/nv-ingest: docs/docs/extraction/nemoretriever-parse.md

!!! important

>     Due to the VRAM usage of nemotron_parse in addition to the other nv-ingest services, it is recommended to run on a [dedicated additional GPU](support-matrix.md). Edit docker-compose.yaml to set nemotron-parse's device_id to a dedicated GPU: device_ids: ["1"] or higher.

Reviewer: ```suggestion
    Due to the VRAM usage of nemotron_parse in addition to the other nv-ingest services, we recommend that you run on a [dedicated additional GPU](support-matrix.md). Edit docker-compose.yaml to set the nemotron-parse `device_id` to a dedicated GPU: device_ids: ["1"] or higher.
```
